src/js/bootstrap.js

```diff
@@ -0,0 +1,4153 @@
+      var a;
+    },
+    findShadowRoot: function (t) {
+      if (!document.documentElement.attachShadow) return null;
+      if ("function" == typeof t.getRootNode) {
+        var e = t.getRootNode();
+        return e instanceof ShadowRoot ? e : null;
+      }
+      return t instanceof ShadowRoot
+        ? t
+        : t.parentNode
+        ? l.findShadowRoot(t.parentNode)
+        : null;
+    },
+    jQueryDetection: function () {
+      if ("undefined" == typeof e)
+        throw new TypeError(
+          "Bootstrap's JavaScript requires jQuery. jQuery must be included before Bootstrap's JavaScript."
+        );
+      var t = e.fn.jquery.split(" ")[0].split(".");
+      if (
+        (t[0] < 2 && t[1] < 9) ||
+        (1 === t[0] && 9 === t[1] && t[2] < 1) ||
+        t[0] >= 4
+      )
+        throw new Error(
+          "Bootstrap's JavaScript requires at least jQuery v1.9.1 but less than v4.0.0"
+        );
+    },
+  };
+  l.jQueryDetection(),
+    (e.fn.emulateTransitionEnd = a),
+    (e.event.special[l.TRANSITION_END] = {
+      bindType: "transitionend",
+      delegateType: "transitionend",
+      handle: function (t) {
+        if (e(t.target).is(this))
+          return t.handleObj.handler.apply(this, arguments);
+      },
+    });
+  var c = "alert",
+    u = e.fn[c],
+    h = (function () {
+      function t(t) {
+        this._element = t;
+      }
+      var n = t.prototype;
+      return (
+        (n.close = function (t) {
+          var e = this._element;
+          t && (e = this._getRootElement(t)),
+            this._triggerCloseEvent(e).isDefaultPrevented() ||
+              this._removeElement(e);
+        }),
+        (n.dispose = function () {
+          e.removeData(this._element, "bs.alert"), (this._element = null);
+        }),
+        (n._getRootElement = function (t) {
+          var n = l.getSelectorFromElement(t),
+            i = !1;
+          return (
+            n && (i = document.querySelector(n)),
+            i || (i = e(t).closest(".alert")[0]),
+            i
+          );
+        }),
+        (n._triggerCloseEvent = function (t) {
+          var n = e.Event("close.bs.alert");
+          return e(t).trigger(n), n;
+        }),
+        (n._removeElement = function (t) {
+          var n = this;
+          if ((e(t).removeClass("show"), e(t).hasClass("fade"))) {
+            var i = l.getTransitionDurationFromElement(t);
+            e(t)
+              .one(l.TRANSITION_END, function (e) {
+                return n._destroyElement(t, e);
+              })
+              .emulateTransitionEnd(i);
+          } else this._destroyElement(t);
+        }),
+        (n._destroyElement = function (t) {
+          e(t).detach().trigger("closed.bs.alert").remove();
+        }),
+        (t._jQueryInterface = function (n) {
+          return this.each(function () {
+            var i = e(this),
+              o = i.data("bs.alert");
+            o || ((o = new t(this)), i.data("bs.alert", o)),
```

Comment:
Codacy found an issue: [A constructor name should not start with a lowercase letter.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)